modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/LimitOffsetTest.java

```diff
@@ -35,15 +35,14 @@
 import org.apache.ignite.internal.util.typedef.F;
 import org.apache.ignite.internal.util.typedef.X;
 import org.apache.ignite.internal.util.typedef.internal.U;
-import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
 import org.junit.Test;
 
 import static java.util.Collections.singletonList;
 
 /**
  * Limit / offset tests.
  */
-public class LimitOffsetTest extends GridCommonAbstractTest {
+public class LimitOffsetTest extends AbstractBasicIntegrationTest {
```

Comment:
Any new integration tests? Or all cases already covered by existing tests? I think we need at least execution node test (see `ExecutionTestSuite`). If there are no new tests in this class, file shouldn't be changed.

Reply:
Reverted